File: Python/Program/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna, untuk, size, merk):
        super().__init__(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna)
        self.untuk = untuk
        self.size = size
        self.merk = merk
    
    def get_untuk(self):
        return self.untuk
    
    def get_size(self):
        return self.size
    
    def get_merk(self):
        return self.merk
    
    def set_untuk(self, untuk):
        self.untuk = untuk
    
    def set_size(self, size):
        self.size = size
    
    def set_merk(self, merk):
        self.merk = merk

File: Java/Program/Baju.java
public class Baju extends Aksesoris {
    private String untuk;
    private String size;
    private String merk;

    public Baju(String id, String namaProduk, double hargaProduk, int stokProduk,
              String jenis, String bahan, String warna,
              String untuk, String size, String merk) {
        super(id, namaProduk, hargaProduk, stokProduk, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    public String getUntuk() {
        return this.untuk;
    }

    public String getSize() {
        return this.size;
    }

    public String getMerk() {
        return this.merk;
    }

    public void setUntuk(String untuk) {
        this.untuk = untuk;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }
}